frontend/src/app.js

```diff
@@ -0,0 +1,48 @@
+
+import 'babel-polyfill/browser';
+import es6Promise from 'es6-promise';
+import 'isomorphic-fetch';
+import 'l20n/dist/compat/web/l20n';
```

Comment:
Looks like this needs to be `import 'l20n';` since [we upgraded L20n](https://github.com/mozilla/testpilot/commit/3681f7a5968785a3118250ff4a128eb38c511009#diff-9e31d7481520a2040f72bb71e10130f1R2)